tests/basic/config.rs

```diff
@@ -3,32 +3,29 @@ use shuttle::sync::Mutex;
 use shuttle::{thread, Config, Runner};
 use std::sync::Arc;
 
-fn check_max_tasks(max_tasks: usize, num_spawn: usize) {
-    let mut config = Config::new();
-    config.max_tasks = max_tasks;
-
-    let scheduler = RandomScheduler::new(100);
+#[test]
+fn many_tasks_with_mutex() {
```

Comment:
Once we fix PCT, we should duplicate this test and run it under PCT as well?

Reply:
👍